### PR TITLE
Changes to "Iterate with JavaScript While Loops - English" challenge description

### LINKED ISSUE #37682: Add more description of while loops to Basic Javascript challenge.
**Describe your problem and how to reproduce it:**  
I feel that the description for the actual code in the [Iterate with JavaScript While Loops - English](https://github.com/Kbarias/freeCodeCamp/blob/master/curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/iterate-with-javascript-while-loops.english.md) challenge should be explained. For example the actual code from the challenge is:

```js
var ourArray = [];
var i = 0;
while(i < 5) {
  ourArray.push(i);
  i++;
}
```

This could possibly be explained as:
The code <code>ourArray.push(i)</code> and <code> i++ </code> will continue to execute while <code>i</code> is less than 5. Thus, this <code>while</code> loop will run 5 times appending the numbers 0 through 4 to <code>ourArray </code>.

### PR DESCRIPTION
**Pull request by Kbarias**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [ X] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [X ] My pull request targets the `master` branch of freeCodeCamp.
- [X ] None of my changes are plagiarized from another source without proper attribution.
- [ X] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [ X] My changes do not use shortened URLs or affiliate links.

Closes #37682

### CODE REVIEW

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/iterate-with-javascript-while-loops.english.md [hunk 1]
```diff
@@ -19,6 +19,7 @@ while(i < 5) {
   i++;
 }
 ```
+The code <code>ourArray.push(i)</code> and <code>i++</code> will continue to execute while <code>i</code> is less than 5. Thus, this <code>while</code> loop will run 5 times appending the numbers 0 through 4 to <code>ourArray</code>.
```
Comment (RandellDawson) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/iterate-with-javascript-while-loops.english.md]: @Kbarias If you click on the Details link, you will see the following error message for the Linting Tests:
> Fenced code blocks should be surrounded by blank lines [Context: "```".  I also recommend a shorter version of the code explanation.

You need to make sure there is an extra line above and below code fence ``` blocks. 
```suggestion

In the code example above, the <code>while</code> loop will execute 5 times and append the numbers 0 through 4 to <code>ourArray</code>.
```

**Review by RandellDawson — CHANGES_REQUESTED** [review]
@ceejayoz See my suggested changes.  You can click the `Commit suggestion` if you do not object to the suggestion.

**Comment by Kbarias** [comment]
Hi @RandellDawson , thank you for the suggested changes. I updated the commit to how you suggested it.

**Review by RandellDawson — APPROVED** [review]

**Review by moT01 — APPROVED** [review]
This LGTM. But I did notice that what the user needs to do for the challenge, is the exact same as the example code. Perhaps we can merge this, and create a new issue around changing either the example or the challenge.

**Comment by RandellDawson** [comment]
@moT01 Open a PR with your suggested change for the challenge instructions and solution.  Sounds like a good change to implement.

**Comment by RandellDawson** [comment]
Congrats @Kbarias on your first pull request (PR)! 🎉

Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 📝

**Pull request merged**